app/Website/src/Http/Requests/SubscribeRequest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Website\Http\Requests;
+
+use Illuminate\Validation\Rule;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rules\RequiredIf;
+
+class SubscribeRequest extends FormRequest
```

Comment:
Please rename this class to `StoreNewsletterSubscriberRequest` to be consistent with the usual Laravel convention.

Reply:
Done